app/views/coronavirus_landing_page/components/hub/_page_header.html.erb

```diff
@@ -44,4 +44,11 @@
     </div>
   </div>
   <%= render 'coronavirus_landing_page/components/hub/guidance_section', guidance: details.guidance_section, related: details.related_links %>
+  <%= render partial: 'components/email-signup-section', locals: {
```

Comment:
Can we have a conversation about the design for this page? I see now why the 'full width' option is present, but even if this design goes ahead I think we could achieve less complexity in the component by doing this a different way.